transactions/transaction_services.py

```diff
@@ -13,8 +13,8 @@
 class TransactionServices:
     """A helper class for online payment services"""
 
-    rave_public_key = os.getenv('RAVE_PUBLIC_KEY')
-    rave_secret_key = os.getenv('RAVE_SECRET_KEY')
+    rave_public_key = os.getenv('RAVE_TEST_PUBLIC_KEY')
```

Comment:
Bi bro, we have agreed to use RAVE_PUBLIC_KEY as variable names, 
Kindly change them in your .env file

Reply:
Thank you @espoirMur